api/types/load_traffic.go

```diff
@@ -77,6 +77,10 @@ type WeightedRequest struct {
 	StaleList *RequestList `json:"staleList,omitempty" yaml:"staleList,omitempty"`
 	// QuorumList means this list request without kube-apiserver cache.
 	QuorumList *RequestList `json:"quorumList,omitempty" yaml:"quorumList,omitempty"`
+	// StaleWatchList means this list request with zero resource version, and it uses streaming list.
+	StaleWatchList *RequestList `json:"staleWatchList,omitempty" yaml:"staleWatchList,omitempty"`
```

Comment:
Just call it `WatchList` instead of `StaleWatchList` and `QuorumWatchList`. It's hard to set revision during testing. So, WatchList in here should fetch latest data from kube-apiserver